courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -1040,270 +1029,225 @@ Declaring Floating Point Types
+  - **Decimal** digits, not bits
 
-* Automatic representation choice
+* Complier choses representation
 
-   - Compiler chooses closest implementation available
-   - May be more accurate, but not less
-   - If not available, declaration is rejected
+  - From **available** floating point types
+  - May be **more** accurate, but not less
+  - If none available |rightarrow| declaration is **rejected**
 
 ---------------------------------
 Predefined Floating Point Types
 ---------------------------------
 
-* Type `Float`
-
-   - Number of digits is implementation-defined
-   - At least 6 digits if the hardware can support 6 or more
+* Type :ada:`Float` ``>= 6`` digits
 
-* Additional implementation-defined types may exist
+* Additional implementation-defined types
 
-   - `Long_Float`, if present, will support at least 11 digits (if...)
+   - :ada:`Long_Float` ``>= 11`` digits
 
-* Use when you don't care what you get
+* General-purpose
+* Best to **avoid** predefined types
 
-   - Or when 6/11/etc. digits will always be enough
-
-* Best to avoid predefined types
-
-   - Portability may suffer
-   - Easier to avoid than Integer
-   - Assume type `Real` is user-defined in class examples
+   - Loss of **portability**
+   - Easy to avoid
 
 ------------------------
 Base Decimal Precision
 ------------------------
 
-* That which is actually provided
+* **Actual** hardware implementation precision
 
-   - Based on type declaration
-   - Perhaps beyond requested number of digits
+   - Based on **type declaration**
+   - May be **better** than requested
 
-* Referenced via attribute `'Base`
+* Attribute :ada:`'Base`
+* Example:
 
-   - Example: Compiler supports 6, 12, or 24 digits of precision
-   - User requests 8 bits of precision
+   - Available: 6, 12, or 24 digits of precision
+   - Type with **8 digits** of precision
 
-   .. code:: Ada
+      .. code:: Ada
 
-      type my_type is digits 8;
+         type My_Type is digits 8;
  
-   - Compiler can create objects using 12 **or** 24 digits of precision
+   - :ada:`My_Type` will have 12 **or** 24 digits of precision
 
 -------------------------------
 Floating Point Type Operators
 -------------------------------
 
-* Ordered by increasing precedence
+* By increasing precedence
 
-   :relational operator: ``= | /= | < | >= | > | >=``
-   :binary adding operator: ``+ | -``
-   :unary adding operator: ``+ | -``
-   :multiplying operator: ``* | /``
-   :highest precedence operator: ``** | abs``
+   :relational operator: :ada:`= | /= | < | >= | > | >=`
+   :binary adding operator: :ada:`+ | -`
+   :unary adding operator: :ada:`+ | -`
+   :multiplying operator: :ada:`* | /`
+   :highest precedence operator: :ada:`** | abs`
 
-* Note on exponentiation
+* *Note* on floating-point exponentiation ``**``
 
-   - Requested power must be an integer value (positive or negative), otherwise you're asking for roots
+   - Power must be :ada:`Integer` (``< 0`` or ``>= 0``)
```

Comment:
So - any integer? If you really want to specify, say positive or negative - but I think removing the parenthesized expression is fine